Java

```diff
@@ -47,34 +58,27 @@ class student{
 		
 		
 		
-		
       	for(int t=0; t<w; t++) {
       	
           
         	System.out.print("Attendance for week # "+(t+1)+": ");
-        	time[t] = s.nextDouble(); 
-        
+        	double hours = s.nextDouble(); 
+        	students[i].addHours(t,hours);
```

Comment:
At the moment Student#addHours() takes a double so unfortunately this will not compile, you will want to do:
`students[i].addHours(hours);`